iFixit/react-commerce
# Refactor product list item type overrides

Another piece of the work being done in #265 

## QA

To QA this let's do a smoke test on product lists with particular focus on how the content of the hero and meta tags is affected when an item type is selected. Try to add an item type override to a product list in Strapi e see how it behaves. The behaviour should be the same as on `main` (you can use https://react-commerce.vercel.app/ and https://main.govinor.com/admin to do the comparison)
